packages/techdocs-common/package.json

```diff
@@ -42,6 +42,7 @@
     "@google-cloud/storage": "^5.6.0",
     "@types/dockerode": "^3.2.1",
     "@types/express": "^4.17.6",
+    "aws-sdk": "^2.813.0",
```

Comment:
It looks like v3 was just released. Is it worth taking a look? Seems like it has better TS support - https://aws.amazon.com/about-aws/whats-new/2020/12/aws-sdk-javascript-version-3-generally-available/.

Reply:
Nice! @ayshiff Would you this this a try? You can create an issue for the upgrade if you want to do it later or want someone else to do it - and that is totally fine.

Reply:
Yes of course ! I'll look into it 👍